internal/io/finder/finder.go

```diff
@@ -0,0 +1,77 @@
+package finder
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/debricked/cli/internal/io/finder/maven"
+)
+
+type IFinder interface {
+	FindMavenRoots(files []string) ([]string, error)
+	FindJavaClassDirs(files []string) ([]string, error)
+	FindFiles(paths []string, exclusions []string) ([]string, error)
+}
```

Comment:
Finder is directly used by callgraph/Generator, maybe we should abstract away the maven/java functionality and make this interface more language agnostic? 

Maybe:
FindRoots
FindDirs
FindFiles